client-app/shared/wishlists/components/wishlist-line-items.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="vc-wishlist-line-items">
+    <!-- table header -->
+    <div class="vc-wishlist-line-items__header">
+      <div class="vc-wishlist-line-items__product">
+        {{ $t("shared.wishlists.wishlist_line_items.product") }}
```

Comment:
Use the localization key `common.labels.product`
Also for the rest of the properties.